### ISSUE TITLE
Bug report: Signing up for a new account can get stuck on "Creating account..."

### ISSUE THREAD

**Issue by BonnieTheCat** [inklewriter/freeinklewriter#39]
* **I'm submitting a ...**
  - [x] bug report
as requested by @albancrommer on Discord :)

* **What is the current behavior?**

I experienced this when attempting to register a new account:

1. Visited https://www.inklewriter.com/
2. Clicked "Sign In" -> "Create New Account".
3. Entered an unregistered (as far as I know!) email address and a password, and clicked "Register".

_Here the pop-up hung on "Creating account..." for a while, then eventually seemed to time out with an error (I didn't write it down, but it was something like "Could not reach server")._

4. Repeated steps 1-3. 

_This time the popup hung for >20mins on "Creating account..."_

5. Clicked "Cancel" -> "Sign In" and entered the email address I attempted to register with.

_Here I got the error "Sorry, could not sign in."_

6. Clicked "Forgotten Password?" -> entered email address -> "Submit".

_This instantly sent me a password recovery email, and I was able to follow that to set the password again and sign in._

* **What is the expected behavior?**
I expected either a confirmation of my account creation after step 3 above, or an error message telling me that my email address was already registered.

* **Please tell us about your environment:**
  
  - Version: 1.0
  - Browser: Firefox 91.0 (32-bit). I also attempted to repeat steps 1-3 on Chrome 92.0.4515.131 (64-bit).

**Comment by albancrommer** [comment]
Won't fix: this was due to databse outage
